media-source-respec.html

```diff
@@ -404,22 +406,18 @@ <h3>Attributes</h3>
               <p class="note">{{SourceBuffer/appendBuffer()}} and {{MediaSource/endOfStream()}} can update the duration under certain circumstances.</p>
             </li>
           </ol>
-        </dd><dt><dfn><code>onsourceopen</code></dfn> of type {{EventHandler}}</dt><dd>
```

Comment:
These are self describing as they are event handlers

Reply:
Respec seems to not like these since they're now rendered in red in the WebIDL, probably due to no reference to or from defining text.

Reply:
Having a look... it should handle these just find as "self defining"